src/Native/Unix/System.Net.Security.Native/pal_gssapi.c

```diff
@@ -21,61 +21,61 @@
 #include <assert.h>
 #include <string.h>
 
-static_assert(PAL_GSS_C_DELEG_FLAG == GSS_C_DELEG_FLAG, "");
-static_assert(PAL_GSS_C_MUTUAL_FLAG == GSS_C_MUTUAL_FLAG, "");
-static_assert(PAL_GSS_C_REPLAY_FLAG == GSS_C_REPLAY_FLAG, "");
-static_assert(PAL_GSS_C_SEQUENCE_FLAG == GSS_C_SEQUENCE_FLAG, "");
-static_assert(PAL_GSS_C_CONF_FLAG == GSS_C_CONF_FLAG, "");
-static_assert(PAL_GSS_C_INTEG_FLAG == GSS_C_INTEG_FLAG, "");
-static_assert(PAL_GSS_C_ANON_FLAG == GSS_C_ANON_FLAG, "");
-static_assert(PAL_GSS_C_PROT_READY_FLAG == GSS_C_PROT_READY_FLAG, "");
-static_assert(PAL_GSS_C_TRANS_FLAG == GSS_C_TRANS_FLAG, "");
-static_assert(PAL_GSS_C_DCE_STYLE == GSS_C_DCE_STYLE, "");
-static_assert(PAL_GSS_C_IDENTIFY_FLAG == GSS_C_IDENTIFY_FLAG, "");
-static_assert(PAL_GSS_C_EXTENDED_ERROR_FLAG == GSS_C_EXTENDED_ERROR_FLAG, "");
-static_assert(PAL_GSS_C_DELEG_POLICY_FLAG == GSS_C_DELEG_POLICY_FLAG, "");
-
-static_assert(PAL_GSS_COMPLETE == GSS_S_COMPLETE, "");
-static_assert(PAL_GSS_CONTINUE_NEEDED == GSS_S_CONTINUE_NEEDED, "");
+c_static_assert(PAL_GSS_C_DELEG_FLAG == GSS_C_DELEG_FLAG);
+c_static_assert(PAL_GSS_C_MUTUAL_FLAG == GSS_C_MUTUAL_FLAG);
+c_static_assert(PAL_GSS_C_REPLAY_FLAG == GSS_C_REPLAY_FLAG);
+c_static_assert(PAL_GSS_C_SEQUENCE_FLAG == GSS_C_SEQUENCE_FLAG);
+c_static_assert(PAL_GSS_C_CONF_FLAG == GSS_C_CONF_FLAG);
+c_static_assert(PAL_GSS_C_INTEG_FLAG == GSS_C_INTEG_FLAG);
+c_static_assert(PAL_GSS_C_ANON_FLAG == GSS_C_ANON_FLAG);
+c_static_assert(PAL_GSS_C_PROT_READY_FLAG == GSS_C_PROT_READY_FLAG);
+c_static_assert(PAL_GSS_C_TRANS_FLAG == GSS_C_TRANS_FLAG);
+c_static_assert(PAL_GSS_C_DCE_STYLE == GSS_C_DCE_STYLE);
+c_static_assert(PAL_GSS_C_IDENTIFY_FLAG == GSS_C_IDENTIFY_FLAG);
+c_static_assert(PAL_GSS_C_EXTENDED_ERROR_FLAG == GSS_C_EXTENDED_ERROR_FLAG);
+c_static_assert(PAL_GSS_C_DELEG_POLICY_FLAG == GSS_C_DELEG_POLICY_FLAG);
+
+c_static_assert(PAL_GSS_COMPLETE == GSS_S_COMPLETE);
+c_static_assert(PAL_GSS_CONTINUE_NEEDED == GSS_S_CONTINUE_NEEDED);
 
 #if !HAVE_GSS_SPNEGO_MECHANISM
 static char gss_spnego_oid_value[] = "\x2b\x06\x01\x05\x05\x02"; // Binary representation of SPNEGO Oid (RFC 4178)
```

Comment:
const?

Reply:
this triggers another instance of `incompatible-pointer-types-discards-qualifiers` at usage and I am not touching it anyway in PR.